### DISCUSSION TITLE
Welcome to nextplayer Discussions!

### DISCUSSION THREAD

**Discussion by anilbeesetti** [anilbeesetti/nextplayer#384] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Maha-Rajan** [comment]
Currently the Quick Setting is utilised minimally with good UI/UX experience but I feel the functionality could be extended a bit more without loosing it's UI/UX... 

I've been using MX player for a long time and I've finally switched to Next player. There are many things Next player could borrow from MX player imo.. I'll point out to the Quick Setting dialogue box... There is an option called fields in the QS dialogue box in MX player that allows to specify what information should be displayed along with the name and thumbnail. This is fairly a simple yet good user functionality that can be implemented. Ex.. I might want to see the frame rate and may be not see the path location... I'll attach a screenshot 
![Screenshot_20230626-180627](https://github.com/anilbeesetti/nextplayer/assets/40870763/783d1595-484a-44dc-9d9f-8f66f58a5683)
for you to look in to it..

**Comment by Maha-Rajan** [comment]
And Can we have search functionality added

**Reply by anilbeesetti** [reply]
will be added in future

**Reply by Maha-Rajan** [reply]
Thanks

**Comment by SaurabhKaperwan** [comment]
Add more aspect ratio

**Comment by Sureshgorai25** [comment]
Can we get delete option and other features like copy/move video from one folder to anothe

**Reply by anilbeesetti** [reply]
sure. will be added soon

**Comment by nobef** [comment]
On my device, the video loading is slow, there is a loading sign. It is also noticeable when rewinding.
  I would also like that the buttons before. play and the next one went down, as in mx, otherwise they close the review. This is bad when watching short videos, while the buttons go away, the floor of the video passes.
  Otherwise, everything is fine, I have long wanted a simple player like mpv, but with a library!

**Comment by Vikranttiw** [comment]
Bro it could not detect a folder in my storage I think because it has many other folder inside and it could not scan it could u please add scaning features a bug check and we couldn't select other folder in scaning option

**Comment by Vikranttiw** [comment]
Hey have you used nova video player it one feature really fasinates me that is it download the nfo files of series and movies and organise files really nicely and can also be used as scoobler for trakt .just asking can it be integrated in next player

**Comment by RYHANium** [comment]
Why don't you upload this app on Play Store? Is there any problem?

**Reply by RYHANium** [reply]
And can you please add the "Check for updates" option in the app's settings?

**Reply by anilbeesetti** [reply]
@RYHANium Next player is available in Play store. Here is the link: https://play.google.com/store/apps/details?id=dev.anilbeesetti.nextplayer

**Comment by alzuobaba** [comment]
and can add function “Long press double speed”

**Comment by VasilisKos** [comment]
I just saw in the Actions tab that you prepare to add a "bottom navigation in media screen" feature. Checked  it in the debug build to see what exactly it is, and I must say I find it pretty useless. Why have all the videos in one huge list? The folder structure is more practical and more than enough. Remember Anil: keep it simple!

So, I'd suggest you remove it before launching the next build. Nevertheless, if you finally decide to keep the feature, I'd suggest to add an option in the settings to choose the format of the media list: "Folder structure" or "Videos list", instead of that bottom navigation. That would save the wasted space in the bottom of the screen.

**Reply by anilbeesetti** [reply]
In the previous version, I encountered difficulty finding room to incorporate a search button and a floating action button for playing the last viewed video. Additionally, I intend to integrate a folder tree into the bottom navigation. Another notable feature to implement is the ability to maintain a history of previously visited web URLs.

**Comment by RYHANium** [comment]
Please give us the option to download subtitles automatically like nova player

**Comment by tm2023-SU** [comment]
Hi,
I am not an expert of formatd. Therefore I have the following question: Does Next Player support 7.1.4 on MP4 files?

Regards

**Comment by VasilisKos** [comment]
Thieves...:
https://play.google.com/store/apps/details?id=com.defaultvideoplayer.videoplayer

**Reply by anilbeesetti** [reply]
@VasilisKos Play store says not found for me

**Reply by VasilisKos** [reply]
Obviously deleted by Google for some reason. Maybe it was just your app with some malware added.  Good for them! 😊

**Reply by themd0** [reply]
why theives? the app is licensed under the GNU I think it's ok for them to publish it

**Reply by VasilisKos** [reply]
Far from licenses, the HONEST and ETHICAL thing is to give credits to the original dev! Not present it as your own work! If not, then you are a thief!

**Reply by themd0** [reply]
It's normal for anyone using the app to brand it, after all the original dev, contriputors and sources are all mentioned in the settings/about.

**Reply by Silther** [reply]
> It's normal for anyone using the app to brand it, after all the original dev, contriputors and sources are all mentioned in the settings/about.  

You still have to ask the author, until then it's just thievery

**Comment by anilbeesetti** [comment]
Hello @VasilisKos and everyone, I hope you're doing well. I could use your assistance in checking whether my PayPal account is set up for receiving international payments. Would you mind sending a small amount to my PayPal? I'll be happy to return it, or if you prefer, I can send it to you first and you can send me back. Thanks for your help!

**Reply by VasilisKos** [reply]
Sure. What's your Paypal account?

**Reply by anilbeesetti** [reply]
Here you go https://www.paypal.me/AnilBeesetti. Thank you

**Reply by VasilisKos** [reply]
Sorry, but would it be easier for you to send something to mine?
paypal.me/VKosmidis
I'll send it back of course. 😊

**Reply by anilbeesetti** [reply]
Okay @VasilisKos. I did

**Reply by anilbeesetti** [reply]
Thanks @VasilisKos

**Reply by VasilisKos** [reply]
I received 0.60€ of the 1 you sent. There was a 0.40€ fee. Just sent you back 0.61$

**Comment by Silther** [comment]
I must say, this app would fit perfectly into the you-apps (https://github.com/you-apps)

**Comment by anoshione** [comment]
I absolutely love the app. Like everyone else I also used to use mx player and for a open source project, this app is already really really good.

And although I would like to see some more feature added to the app, I would say next player is already amazing and I'd love to see how the player grows in the future.
